README.md

```diff
@@ -80,6 +84,39 @@ For more info on providers, please refer to [this](https://github.com/pystardust
 
 ### Linux
 
+### Debian
+
+```
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/KEY.gpg | sudo tee /etc/apt/trusted.gpg.d/ani-cli.asc
+wget -qO- https://Wiener234.github.io/ani-cli-ppa/ani-cli-debian.list | sudo tee /etc/apt/sources.list.d/ani-cli-debian.list
+sudo apt update
+```
+
+### Fedora
+
+To install mpv (and vlc) you need _RPM Fusion free_ enabled. Simply follow the instructions here: https://rpmfusion.org/Configuration
+To be able to install syncplay, you'll need to enable this copr repo: https://copr.fedorainfracloud.org/coprs/batmanfeynman/syncplay/
+
+There are currently two packages for Fedora: `ani-cli` which is a minimal installation that can only use mpv and `ani-cli-full` which contains all the functionality (note that vlc and syncplay are still optional dependencies).
+
+To install ani-cli or ani-cli-full:
+```sh
+sudo dnf copr enable derisis13/ani-cli
+sudo dnf install ani-cli      # only for ani-cli
+sudo dnf install ani-cli-full # only for ani-cli-full
+```
+*If for your distro uses rpm and you would like to see a native package, open an issue.*
+
+### Arch
+
+Also consider ani-cli-git
+
+```sh
+yay -S ani-cli
+```
+
+### Installing from source
```

Comment:
Maybe give nix os a section too, they have a package thats well maintained iirc

Reply:
Ok. Do we know who the nixpkg maintainer is?